pkg/apis/sources/v1alpha1/mongodbsource_defaults_test.go

```diff
@@ -23,28 +23,23 @@ import (
 	"github.com/google/go-cmp/cmp"
 )
 
-func TestCouchDbDefaults(t *testing.T) {
+func TestMongoDbDefaults(t *testing.T) {
 	testCases := map[string]struct {
-		initial  CouchDbSource
-		expected CouchDbSource
+		initial  MongoDbSource
+		expected MongoDbSource
 	}{
 		"nil spec": {
-			initial: CouchDbSource{},
-			expected: CouchDbSource{
-				Spec: CouchDbSourceSpec{
-					Feed: FeedContinuous,
-				},
+			initial: MongoDbSource{},
+			expected: MongoDbSource{
+				Spec: MongoDbSourceSpec{},
 			},
 		},
 		"feed not set": {
```

Comment:
this name does not apply here, we do not have a feed value. 
once you change the service account to be "default", then you will be able to test that here. We should do so